vllm/model_executor/sampling_metadata.py

```diff
@@ -94,12 +94,18 @@ def __init__(
         selected_token_indices: torch.Tensor,
         categorized_sample_indices: Dict[SamplingType, torch.Tensor],
         num_prompts: int,
+        skip_sampler_cpu_output: bool = False,
+        reuse_sampling_tensors: bool = False,
     ) -> None:
         self.seq_groups = seq_groups
         self.selected_token_indices = selected_token_indices
         self.categorized_sample_indices = categorized_sample_indices
         self.num_prompts = num_prompts
 
+        # TODO: Add docs
```

Comment:
Remind: add docstring and remove this comment

Reply:
Good catch, added